Client/mods/deathmatch/logic/CClientPrimitiveBufferManager.cpp

```diff
@@ -0,0 +1,81 @@
+/*****************************************************************************
+ *
+ *  PROJECT:     Multi Theft Auto
+ *               (Shared logic for modifications)
+ *  LICENSE:     See LICENSE in the top level directory
+ *  FILE:        mods/shared_logic/CClientPrimitiveBufferManager.cpp
+ *  PURPOSE:     Primitive buffer manager class
+ *
+ *****************************************************************************/
+
+#include "StdInc.h"
+
+using std::list;
+
+CClientPrimitiveBufferManager::CClientPrimitiveBufferManager(CClientManager* pManager)
+{
+    m_pManager = pManager;
+}
+
+CClientPrimitiveBufferManager::~CClientPrimitiveBufferManager()
+{
+    DeleteAll();
+}
+
+CClientPrimitiveBuffer* CClientPrimitiveBufferManager::Get(unsigned long ulID)
+{
+    // Find the PrimitiveBuffer with the given id
+    list<CClientPrimitiveBuffer*>::const_iterator iter = m_List.begin();
```

Comment:
You could just use a range based for here:
```
for(conat auto& iter : m_List) 
{
    if(iter->GetID() == ulID)
       return iter;
}
return nullptr;
```

Reply:
is `conat` some new c++ 20 standard keyword which speed up everything by 0.0001% ? i'm joking ofc

Reply:
const doesnt do anything, well, nearly anything. It mostly is for `visuals`, that is, it indicates, that the data wont be changed. Here you can't use it, since im pretty sure `GetID()` isn't `const`, so, just use `auto& iter`.